pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -590,13 +595,13 @@ func (nc *Controller) doNodeProcessingPassWorker(ctx context.Context) {
 		}
 		nodeName := obj.(string)
 		if err := nc.doNoScheduleTaintingPass(ctx, nodeName); err != nil {
-			klog.Errorf("Failed to taint NoSchedule on node <%s>, requeue it: %v", nodeName, err)
+			logger.Error(err, "Failed to taint NoSchedule on node, requeue it", "nodeName", nodeName)
```

Comment:
```suggestion
			logger.Error(err, "Failed to taint NoSchedule on node, requeue it", "node", klog.KRef("", nodeName))
```

For consistency. Please replace in the entire PR.

Instead of importing "github.com/go-logr/logr", continue to import "klog" and use "klog.Logger" if you need the type - it's an alias that was added for that purpose.

Reply:
done